saleor/graphql/core/utils/error_codes.py

```diff
@@ -0,0 +1,169 @@
+from enum import Enum
+
+DJANGO_VALIDATORS_ERROR_CODES = [
+    "invalid",
+    "invalid_extension",
+    "limit_value",
+    "max_decimal_places",
+    "max_digits",
+    "max_length",
+    "max_value",
+    "max_whole_digits",
+    "min_length",
+    "min_value",
+    "null_characters_not_allowed",
+]
+
+DJANGO_FORM_FIELDS_ERROR_CODES = [
+    "contradiction",
+    "empty",
+    "incomplete",
+    "invalid_choice",
+    "invalid_date",
+    "invalid_image",
+    "invalid_list",
+    "invalid_time",
+    "missing",
+    "overflow",
+    "required",
+]
+
+DJANGO_PASSWORD_VALIDATION_ERROR_CODES = [
+    "password_entirely_numeric",
+    "password_too_common",
+    "password_too_short",
+    "password_too_similar",
+]
+
+DJANGO_MODEL_FIELDS_ERROR_CODES = ["blank", "null", "unique", "unique_for_date"]
+
+
+class AccountErrorCode(Enum):
+    ACTIVATE_OWN_ACCOUNT = "activate_own_account"
+    ACTIVATE_SUPERUSER_ACCOUNT = "activate_superuser_account"
+    DEACTIVATE_OWN_ACCOUNT = "deactivate_own_account"
+    DEACTIVATE_SUPERUSER_ACCOUNT = "deactivate_superuser_account"
+    DELETE_NON_STAFF_USER = "delete_non_staff_user"
+    DELETE_OWN_ACCOUNT = "delete_own_account"
+    DELETE_STAFF_ACCOUNT = "delete_staff_account"
+    DELETE_SUPERUSER_ACCOUNT = "delete_superuser_account"
+    INVALID_COUNTRY = "invalid_country"
+    INVALID_PASSWORD = "invalid_password"
+    INVALID_PHONE_NUMBER = "invalid_phone_number"
+    NOT_USERS_ADDRESS = "not_users_address"
```

Comment:
Why is this useful? Do we ever need the UI to know that an address ID is correct but belongs to a different user?